docs/en/latest/plugins/server-info.md

```diff
@@ -41,6 +41,12 @@ The information reported by the Plugin is explained below:
 | version      | string  | Version of APISIX instance.                                                                                            |
 | hostname     | string  | Hostname of the machine/pod APISIX is deployed to.                                                                     |
 
+:::warning
+
+This plugin won't work if the APISIX role is `data_plane`.
```

Comment:
We don't need a notice as there is always a report from CP or traditional role.

Reply:
rm it